ios/Surveys/Sources/Presentation/Modules/Home/HomeView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  HomeView.swift
+//  Surveys
+//
+//  Created by Su Ho on 03/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct HomeView: View {
+    var body: some View {
+        VStack {
+            HomeHeaderView(
+                title: "Monday, JUNE 16",
+                imageURLString: "https://cdn.pixabay.com/photo/2016/11/18/23/38/child-1837375__340.png"
+            )
+            .background(Color.gray) // Remove this later
+            .padding(EdgeInsets(top: 0.0, leading: 20.0, bottom: 0.0, trailing: 20.0))
+            Spacer()
+        }
+    }
+}
+
+struct HomeView_Previews: PreviewProvider {
+    static var previews: some View {
```

Comment:
same here.
```suggestion
struct HomeView_Previews: PreviewProvider {

    static var previews: some View {
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/97/commits/53bd127fdd575af6b3e1af2f8515a6b72a02f44b